sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/EventGridPublisherClientImpl.java

```diff
@@ -115,10 +115,10 @@ public SerializerAdapter getSerializerAdapter() {
      * The interface defining all the services for EventGridPublisherClient to be used by the proxy service to perform
      * REST calls.
      */
-    @Host("https://{topicHostname}")
+    @Host("{topicHostname}")
     @ServiceInterface(name = "EventGridPublisherCl")
     private interface EventGridPublisherClientService {
-        @Post("/api/events")
```

Comment:
Do we have a tracking issue to fix the swagger?

Reply:
Followup issue https://github.com/Azure/azure-sdk-for-java/issues/19769